.github/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a bug report to help us improve QGIS.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Checklist before submitting**
+
+
+- [ ] Search in the issues, stackexchange, forums before if the issue already exists
+- [ ] Test with a [clean new user profile](https://docs.qgis.org/testing/en/docs/user_manual/introduction/qgis_configuration.html?highlight=profile#working-with-user-profiles). Do not add third party plugins at that step
+- [ ] Create light and portable sample dataset and project file that reproduce the issue
+
+If the issue concerns a **third party plugin**, please raise the issue in the dedicated bug tracker mentionned in plugin description.
+
+**Describe the bug**
+A clear and concise description of what the bug is. Tip
```

Comment:
IMHO, there is a lot of strings to remove when we fill a bug. I guess we prefer not having these strings in the final report. Can we put this help as comment? But comments are also harder for beginners.
Check the result and the raw file of https://github.com/inasafe/inasafe/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Good point, I used the default github template to start. I'll try to simplify all this

Reply:
So it seems you are not going to use comments 😉

Reply:
> So it seems you are not going to use comments wink

In fact I didn't get what you mean Etienne, can you be more explicit?

Reply:
You can put strings as markdowns comments. They are not displayed by default. Only when editing the markdown. Check my link how it's displayed and check the raw file. You will see some markdowns comments only when you click on the raw file. It's much much better IMHO because these strings are not displayed when we read the issue, only when we edit it. So it's less "noise" for readers.

Reply:
@Gustry understood now :) ID didn't see it directly in your link as I didn't switch to edit mode . 
Let's do that !

Reply:
Done